tutorials/in-app/cameraParallax.json

```diff
@@ -0,0 +1,750 @@
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Let's select this layer as the **default layer** to add objects to.",
+            "fr": "Sélectionnons ce calque comme **calque par défaut** pour ajouter des objets.",
+            "es": "Seleccionemos esta capa como la **capa predeterminada** para agregar objetos.",
+            "pt": "Vamos selecionar esta camada como a **camada padrão** para adicionar objetos."
+          }
+        },
+        "placement": "left"
+      }
+    },
+    {
+      "elementToHighlightId": "#layer-1 #layer-drag-handle",
+      "nextStepTrigger": {
+        "presenceOfElement": "#layer-0 #layer-selected-checked"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Now let's **move** this layer **below the Base Layer**, so that the background elements will appear behind the player.\n\n**Drag the handle below**",
+            "fr": "Maintenant, **déplaçons** ce calque **en dessous du calque de base**, de sorte que les éléments de fond apparaissent derrière le joueur.\n\n**Glissez le curseur en dessous**",
+            "es": "Ahora, **mueva** esta capa **debajo de la capa base**, para que los elementos de fondo aparezcan detrás del jugador.\n\n**Arrastre el control hacia abajo**",
+            "pt": "Agora, **mova** esta camada **abaixo da camada base**, para que os elementos de fundo apareçam atrás do jogador.\n\n**Arraste o controle para baixo**"
+          }
+        },
+        "placement": "top"
+      }
+    },
+    {
+      "elementToHighlightId": "objectInObjectsList:farBackground",
+      "nextStepTrigger": {
+        "instanceAddedOnScene": "farBackground"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Perfect! Now let's add **$(farBackground)** to our game by dragging it to the center.",
+            "fr": "Parfait ! Ajoutons maintenant **$(farBackground)** à notre jeu en le faisant glisser au centre.",
+            "es": "¡Perfecto! Ahora agreguemos **$(farBackground)** a nuestro juego arrastrándolo al centro.",
+            "pt": "Perfeito! Agora vamos adicionar **$(farBackground)** ao nosso jogo, arrastando-o para o centro."
+          }
+        },
+        "placement": "left"
+      }
+    },
+    {
+      "nextStepTrigger": {
+        "clickOnTooltipButton": {
+          "messageByLocale": {
+            "en": "I'm done",
+            "fr": "J'ai terminé",
+            "es": "Ya terminé",
+            "pt": "Terminei"
+          }
+        }
+      },
+      "tooltip": {
+        "standalone": true,
+        "description": {
+          "messageByLocale": {
+            "en": "See the **black square** in the middle of the scene? This is the **camera view**. It's the point of view from which the player will see the game.\n\nLet's resize the **$(farBackground)** object so that it covers the whole black square\n\nWhen you're done, click below.",
```

Comment:
You could add a base64 image to show how to do it!

Reply:
added! nice addition indeed